itest/client.go

```diff
@@ -216,11 +217,8 @@ func (c *Client) CreateAccount(creator *Account, name string, key *Key, check bo
 func (c *Client) ContractTransfer(cid string, sender, recipient *Account, amount string, memoSize int, check bool) error {
 	memo := make([]byte, memoSize)
 	rand.Read(memo)
-	// Convert to base64.
-	memo, _ = json.Marshal(memo)
-	// Get rid of the beginning quote and cut to the string of given size.
-	memo = memo[1 : memoSize+1]
-	_, err := c.CallAction(check, sender, cid, "transfer", sender.ID, recipient.ID, amount, string(memo))
+	memoStr := base64.StdEncoding.EncodeToString(memo)
+	_, err := c.CallAction(check, sender, cid, "transfer", sender.ID, recipient.ID, amount, memoStr)
 	return err
```

Comment:
The size of this memoStr will not be the size we expected with memoSize. Because base64 encoding will increase the size of memo.

Reply:
I'll fix that soon.